pravega-client-examples/src/main/java/io/pravega/example/secure/SecureReader.java

```diff
@@ -0,0 +1,186 @@
+        try {
+            ReaderGroupConfig readerGroupConfig = ReaderGroupConfig.builder()
+                    .stream(Stream.of(this.scope, this.stream))
+                    .disableAutomaticCheckpoints()
+                    .build();
+            System.out.println("Done creating a reader group config with specified scope: [" +
+                    this.scope +"] and stream name: [" + this.stream + "].");
+
+            String readerGroupName = UUID.randomUUID().toString().replace("-", "");
+            readerGroupManager = ReaderGroupManager.withScope(this.scope, clientConfig);
+            readerGroupManager.createReaderGroup(readerGroupName, readerGroupConfig);
+            System.out.println("Done creating a reader group with specified name  and config.");
+
+            clientFactory = ClientFactory.withScope(this.scope, clientConfig);
+            System.out.println("Done creating a client factory with the specified scope and client config.");
+
+            reader = clientFactory.createReader("readerId", readerGroupName,
+                    new JavaSerializer<>(), ReaderConfig.builder().build());
+            System.out.println("Done creating a reader.");
+
+            String readMessage = reader.readNextEvent(2000).getEvent();
+            System.out.println("Done reading an event: [" + readMessage + "].");
+
+        } finally {
+            if (reader != null) reader.close();
+            if (clientFactory != null) clientFactory.close();
+            if (readerGroupManager != null) readerGroupManager.close();
+        }
+        System.err.println("All done with reading! Exiting...");
+    }
+
+    public static void main(String[] args) throws ReinitializationRequiredException {
+        Options options = getOptions();
+
+        CommandLine cmd = null;
+        try {
+            cmd = parseCommandLineArgs(options, args);
+        } catch (ParseException e) {
+            System.out.format("%s.%n", e.getMessage());
+            final HelpFormatter formatter = new HelpFormatter();
+            formatter.printHelp("HelloWorldReader", options);
+            System.exit(1);
+        }
+
+        final String scope = cmd.getOptionValue("scope") == null ? Constants.DEFAULT_SCOPE : cmd.getOptionValue("scope");
+        final String stream = cmd.getOptionValue("stream") == null ? Constants.DEFAULT_STREAM_NAME : cmd.getOptionValue("stream");
+
+        final String uriString = cmd.getOptionValue("uri") == null ? Constants.DEFAULT_CONTROLLER_URI : cmd.getOptionValue("uri");
+        final URI controllerURI = URI.create(uriString);
+
+        final String truststorePath =
+                cmd.getOptionValue("truststore") == null ? Constants.DEFAULT_TRUSTSTORE_PATH : cmd.getOptionValue("truststore");
+        final boolean validateHostname = cmd.getOptionValue("validatehost") == null ? false : true;
+
+        final String username = cmd.getOptionValue("username") == null ? Constants.DEFAULT_USERNAME : cmd.getOptionValue("accountname");
```

Comment:
It uses both `username` and `accountname`? Also the naming convention is not camelCase.

Reply:
My initial thought was using accountname as an external abstraction and username as an internal one, as:
* When using a client program, it's not really a user which is the subject of the authentication, but rather a program using a certain account. 
* Username is what we use internally in the code everywhere. 

As per your suggestion, I modified it to `username`. It keeps things much simpler. 

Also, w.r.t. came casing, `username` is a standard noun (witness https://www.dictionary.com/browse/username, for example). So, it doesn't require camel casing as it is a single word.